### ISSUE TITLE
retry button on http error api/message/short (viewing pwd encr msg)

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#1444]

**Comment by tomholub** [comment]
FlowCrypt has a way to send msgs to users who don't have encryption software. Encrypted with a password, then uploaded to our servers, then served to recipient through a link. Archaic blog post https://flowcrypt.com/blog/article/cryptup-just-got-a-paid-feature-for-free

When someone receives one like that and later installs FlowCrypt, these messages get downloaded to the browser to decrypt and render them. But the http request can fail due to network issues, server issues, etc.

What we need is showing what happened and a retry button. For this look up the `Api.err.eli5` method that will convert an exception into a human-readable failure reason. Show that and add a retry button that reloads the iframe, which causes a retry.

**Comment by tomholub** [comment]
Reproduce:
1) from mainacct@gmail.com send email to someacctthatdoesnthaveflowcryptinstalled@gmail.com - and when you are prompted that there is no pubkey, choose the one-time-password option
2) install FlowCrypt at the second account
3) try to open the message
4) it should open with a password. Now, set browser to offline and reload the frame
5) It will probably error out in some ugly way. This is where we need a retry button.

Hint: to send more messages to an acct that has already been set up, if it's on the @gmail domain, you can abuse dots and it will end up in the same mailbox. myself@gmail.com === my.self@gmail.com === mys.elf@gmail.com. All the other addresses will appear like not having

Hint: retries are a common problem. Search for 'retry' in the code and you'll be sure to find some code to re-use, likely some code that reloads the iframe in the same manner.

Your work will likely be centered somewhere around `elements/pgp_block.ts`

**Comment by tomholub** [comment]
Correction, the decrypt button seems to do nothing... bug. Maybe you can find out why!

**Comment by tomholub** [comment]
Bonus points if the "Wrong password" text could be made red color.

![image](https://user-images.githubusercontent.com/6306961/52441569-e56f5900-2b18-11e9-902c-5c35ebfd56bd.png)

**Comment by evanhellmuth** [comment]
Claiming this issue.
